backend/credentials.json

```diff
@@ -0,0 +1 @@
+{"installed":{"client_id":"926735790990-7v461i3mtc9kp2st49oc8thvd6of94jj.apps.googleusercontent.com","project_id":"quickstart-1571873812130","auth_uri":"https://accounts.google.com/o/oauth2/auth","token_uri":"https://oauth2.googleapis.com/token","auth_provider_x509_cert_url":"https://www.googleapis.com/oauth2/v1/certs","client_secret":"_Hg0BitokaIuOb3WDZo_BN5x","redirect_uris":["urn:ietf:wg:oauth:2.0:oob","http://localhost"]}}
```

Comment:
Please search up how to add this file and `token.js` to the existing .gitignore file so that these files are not committed to Github and only stored locally. These files could allow anyone who has access to use our google account.

Reply:
*token.json